ktor-server/ktor-server-plugins/ktor-server-cors/common/src/io/ktor/server/plugins/cors/CORSUtils.kt

```diff
@@ -135,3 +186,7 @@ internal fun normalizeOrigin(origin: String): String {
 
     return builder.toString()
 }
+
+internal inline fun Logger.info(message: () -> String) {
+    info(message())
```

Comment:
Should we add `if (isInfoEnabled)` condition to save memory allocations as it was done for `trace {}`?

Reply:
That will require publishing a new API and postponing the fix until 3.4.0. So, what's better: to leave the current logging as is and accept the allocation costs, or use the TRACE level?